app/routes/ffxiv.best-deals._index.tsx

```
@@ -0,0 +1,257 @@
+              inputMap[field as keyof BestDealsProps] || 'Unknown input error'
+          )
+        )
+        .join(', ')}`
+    })
+  }
+
+  const result = await FFXIVBestDeals(validatedFormData.data)
+  return json(await result.json())
+}
+
+type ActionResponseType = {} | { exception: string } | BestDealsResponse
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)